### PR TITLE
Correct sensor descriptions

### LINKED ISSUE #40: id_log diagnostics not recorded correctly
### The problem

The `id_log` key in HA diagnostics should display raw and localized values for `stateProgramPhase`, `stateProgramID` and `stateProgramType`. In practice, `stateProgramPhase` appears to be the only logged `key`, with `raw` value consistent with with that `key`. However, `localized` is clearly from `stateProgramID`.

### Version of miele integration having the issue?

miele-0.0.11

### Version of Home Assistant Core having the issue?

core-2022.2.6

### Anything in the logs that might be useful for us?

```txt
"id_log": [
      {
        "appliance": "Dishwasher",
        "key": "stateProgramPhase",
        "raw": 1792,
        "localized": "Automatic"
      },
      {
        "appliance": "Washer dryer",
        "key": "stateProgramPhase",
        "raw": 256,
        "localized": ""
      },
      {
        "appliance": "Dishwasher",
        "key": "stateProgramPhase",
        "raw": 1792,
        "localized": "Automatic"
      },
      {
        "appliance": "Washer dryer",
        "key": "stateProgramPhase",
        "raw": 256,
        "localized": ""
      },
      {
        "appliance": "Dishwasher",
        "key": "stateProgramPhase",
        "raw": 1792,
        "localized": "Automatic"
      },
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mundschenk-at**
Fixes #40.

### CODE REVIEW

**Review by astrandb — APPROVED** [review]
Good catch!
Tank you.

File: custom_components/miele/sensor.py [hunk 1]
```diff
@@ -371,7 +371,7 @@ class MieleSensorDefinition:
         description=MieleSensorDescription(
             key="stateProgramPhase",
             data_tag="state|programPhase|value_raw",
-            data_tag_loc="state|ProgramID|value_localized",
+            data_tag_loc="state|programType|value_localized",
```
Comment (astrandb) [inline custom_components/miele/sensor.py:374]:
```suggestion
            data_tag_loc="state|programPhase|value_localized",
```

**Review by astrandb — CHANGES_REQUESTED** [review]
A little glitch sneaked in...

**Review by astrandb — APPROVED** [review]

**Comment by mundschenk-at** [comment]
> A little glitch sneaked in...

Yeah, looks like it was bit late. I corrected it locally in my HA instance and then prepared the PR, producing another c&p issue while fixing one 🥴

**Pull request merged**